services/users.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+const express = require('express');
+const { Client } = require('pg');
+const router = express.Router();
+const env = require('dotenv').config();
+
+function getUser (username) {
+  const client = new Client({
+    connectionString: process.env.HEROKU_POSTGRESQL_PINK_URL
+  });
+
+  client.connect();
+  return client.query(`SELECT * FROM users WHERE username = '${username}'`);
```

Comment:
By returning the client query, does this then close the client connection? If not, should we not close the client connection before spinning up any new clients?

Reply:
@christopher-gordon98 You should not be worried about closing connection at the the moment,  postgres will drop unused connection after a certain time.. as well as don't wind yourself up with all this stuff there are lots more stuff you need to go through before we even discuss that

Reply:
I'm happy to merge without client disconnects, we're small scale enough that it won't matter for now